### ISSUE TITLE
Accessible designs for notifications & inline messages

### ISSUE THREAD

**Issue by Stan-vw** [piwik/piwik#20691]
Problem: (some of) the current notifications / banners / inline messages aren’t WCAG compliant 
Solution: create & implement on-brand WCAG compliant designs for each of the elements

Actions:
1. List all relevant elements (done)
2. Create new designs (done)
3. Implement designs

Component: Notifications
1. Informative
2. Success
3. Warning
4. Error (alert)

Analysis
There are four types of notifications (as mentioned above) that are commonly used across products, the fonts used in them should have enough contrast (4.5:1 minimum requirement) to comply with accessibility standards and same goes with UI elements (3:1).

We currently have these notifications but they don’t fully comply with accessibility and have different variations of these which are not being used consistently across the product, some can be found on github and some are not there.

Proposed solution
Notifications:
- Notifications are unified = consistency → we don’t have two different versions for each type
- Colours are now accessible
- black font
- light shade of colour in the background 
- icon and border comply with accessibility standards for UI element 
- if notification has a subject, it goes in bold → hierarchy

Inline messages:
- Colours are now accessible
- Dark grey font
- light shade of grey in the background 

![image](https://github.com/matomo-org/matomo/assets/52564377/75d4d5fb-fd96-40fd-ad2d-b99049d12b27)
![image](https://github.com/matomo-org/matomo/assets/52564377/3ae2025c-3a1f-417a-a3cd-0579ae8be081)
![image](https://github.com/matomo-org/matomo/assets/52564377/662366a3-2861-4fd8-aacc-fd1d73afacfb)
![image](https://github.com/matomo-org/matomo/assets/52564377/7cf6722d-4ab6-46b8-8669-80b8c2ea9f36)
![image](https://github.com/matomo-org/matomo/assets/52564377/633e5b66-e5cc-4b0a-b433-711e20dc2087)
![image](https://github.com/matomo-org/matomo/assets/52564377/13901ba2-b50b-4035-aa22-ef7a2796ef5f)
![image](https://github.com/matomo-org/matomo/assets/52564377/b238be92-026e-4eb4-8181-2e802ef8c9d4)
![image](https://github.com/matomo-org/matomo/assets/52564377/1e3604be-dd72-42d4-bb09-13e0330d41b8)
![image](https://github.com/matomo-org/matomo/assets/52564377/1829b0d7-9054-4db8-9413-4e0b145c414d)
![image](https://github.com/matomo-org/matomo/assets/52564377/f4c99c18-1520-442f-ad42-e240269e9d89)
![image](https://github.com/matomo-org/matomo/assets/52564377/2406630d-8ece-47ab-b046-1e9e3b716894)
